src/thor/bidirectional_astar.cc

```diff
@@ -1270,7 +1270,7 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
     // recost edges in final path; ignore access restrictions
     try {
       sif::recost_forward(graphreader, *costing_, edge_cb, label_cb, source_pct, target_pct,
-                          time_info, invariant, true);
+                          time_info, invariant && options.invariant_postprocess(), true);
```

Comment:
so based on my previous comments, here for example we would change it to something like this:

```suggestion
                          time_info, options.has_date_time_type() ? options.date_time_type() : boost::none, true);
```